.gitignore

```diff
@@ -19,6 +19,10 @@
 # Various temporary generated files
 *.tmp
 
+# editor includes
```

Comment:
See the comment at the top of this file: IDE specific files should be in a local .gitignore.